Terraform_modules/ec2/main.tf

```diff
@@ -0,0 +1,64 @@
+data "aws_ami" "ubuntu" {
```

Comment:
why did you combine all instances in a single module?

Reply:
> why did you combine all instances in a single module?

already update everything, review please